spec/index.html

```diff
@@ -253,25 +253,43 @@
     <section>
       <h2>Introduction</h2>
       <p>
-        JSON-LD is a JSON-based format to serialize Linked Data.
-        In recent years, YAML emerged as a more concise format
-        to represent information serialized as JSON,
+        [[JSON-LD]] is a JSON-based format to serialize Linked Data.
+        In recent years, [[YAML]] emerged as a more concise format
+        to represent information serialized as [[JSON]],
         including API specification, data schemas, and Linked Data.
       </p>
       <p>
         Since YAML is more expressive than JSON,
         both in the available data types and in the document structure
         (see [[I-D.ietf-httpapi-yaml-mediatypes]]),
         this document identifies constraints on YAML
-        such that it can be used to represent JSON-LD documents.
+        such that any <a>YAML-LD document</a> can be represented in JSON-LD.
       </p>
     </section>
 
   <section id="conformance">
-    <p>A <a>YAML-LD document</a> complies with this specification if ...</p>
-    <p class="ednote">Define <dfn>YAML-LD document</dfn> somewhere.</p>
+    <h2>YAML-LD stream</h2>
+
+    <p>
+      <dfn>YAML-LD stream</dfn> is a <a href="https://yaml.org/spec/1.2.2/#92-streams">YAML stream</a>
+      of <a>YAML-LD documents</a>.
+    </p>
+
+    <p class="note" title="Documents in the stream are independent of each other">
+        Each document in a stream has its own context, [[YAML]] directives, named anchors, and so on.
+    </p>
+
+    <h2>YAML-LD document</h2>
+
     <p>
-      The term media type is imported from [[RFC6838]].
+      Under a <dfn>YAML-LD document</dfn>, we understand any [[YAML]] document that, after a conversion to [[JSON]],
+      is a valid [[JSON-LD]] document and can be interpreted as [[LINKED-DATA]].
+    </p>
+
+    <h2>Terms</h2>
+
+    <p>
+      The term "media type" is imported from [[RFC6838]].
```

Comment:
These term references should probably be in the form `<a data-dfn="RFC6838#media-type">media type</a>`.

I've offered to redo this along the lines of the JSON-LD docs, so they can be `<dfn data-dfn="RFC6838#media-type">media type</dfn>`, which has the same effect but allows other term references to simply be <a>media type</a>, for example.

Reply:
I didn't know this form, thank you for adding these links.